### PR TITLE
Replacing http:// with https://

### PR DESCRIPTION
**Pull request by dfa1**
# What problem does this pull request solve?
Switching to HTTPS for most URLs

# Please provide any additional information below.

# NOTE
* Please run `mvn spotless:apply` to format the code before opening a PR. Otherwise, GitHub Actions will complain at you 😉. Unfortunately, you will need to have Node installed to do so.
* Mutation tests will be run by [PITest](https://pitest.org/) after opening the PR. It will post comments in the PR for each issue found. Please take a look at them, but if the comments don't make sense, please don't worry about them.

### CODE REVIEW

**Comment by jqno** [comment]
Sorry for the late reply.
Thanks for this PR! I never thought to do this, especially the ones in the Java code itself...

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <name>EqualsVerifier</name>
     <version>3.8.2-SNAPSHOT</version>
     <description>EqualsVerifier can be used in JUnit unit tests to verify whether the contract for the equals and hashCode methods is met.</description>
-    <url>http://www.jqno.nl/equalsverifier</url>
+    <url>https:///www.jqno.nl/equalsverifier</url>
```
Comment (TobiX) [inline pom.xml:10]: Too many slashes?

Comment (dfa1) [reply pom.xml:10]: good catch!

Comment (jqno) [reply pom.xml:10]: Indeed, good catch! Since this PR is already merged, I'll fix it on `main` directly.

**Pull request merged**